pegasus/sites.v3/hourofcode.com/public/css/front-page.css

```diff
@@ -21,3 +21,19 @@
   padding: 20px 4px 0px 4px;
 }
 
+.category-header {
+  font-family: 'Gotham 5r', sans-serif;
+}
+
+.category-button {
+  background-color: #a69bc1;
```

Comment:
I feel like it's arbitrary to set the default to be purple and then override the others with different colors. I think the default should be a gray or charcoal, and then create additional classes for the other colors. That will also replace the in-line css.

    .category-button {
      border-radius: 5px;
      color: white;
      font-family: 'Gotham 4r', sans-serif;
      margin: 5px 0px;
      padding: 8px;
    }

    .purple-background {
      background-color: #a69bc1;
    }

    %div.category-button.purple-background